ckcp/openshift_dev_setup.sh

```diff
@@ -308,7 +314,7 @@ install_compute() {
     "$GITOPS_DIR/pac/setup.sh"
 
   echo "  - Register compute to KCP"
-  KCP_ORG="root:default" KCP_WORKSPACE="$kcp_workspace" DATA_DIR="$WORK_DIR" KCP_SYNC_TAG="$kcp_version" \
+  KCP_ORG="$kcp_org" KCP_WORKSPACE="$kcp_workspace" DATA_DIR="$WORK_DIR" KCP_SYNC_TAG="$kcp_version" \
```

Comment:
probably due to register.sh but I have hundreds of contexts in my home kubeconfig and I get thousand of lines displayed. Also I get two synctargets created, one not be relevant (for a dead cluster)

Reply:
Too much output: caught and fixed in #151 (see the added -q to grep on line 160).

Regarding the 2 synctargets, I'll try to see if I can reproduce and fix.

Reply:
Fix for too many targets added to #151.